user-guide/Troubleshooting/Known_issues/Known_issues.md

```diff
@@ -16,6 +16,7 @@ uid: Known_issues
 | [Corrupted low-code app after concurrent editing actions](xref:KI_app_corruption_after_editing) | From DataMiner 10.2.5/10.3.0 onwards | | May 26, 2023 |
 | [NATS not starting if DMS name contains special characters](xref:KI_NATS_not_starting_special_chars) | From DataMiner 10.1.0/10.1.2 <br>onwards | | November 8, 2022 |
 | [Upgrade fails because of VerifyClusterPort.dll prerequisite](xref:KI_Upgrade_fails_VerifyClusterPorts_prerequisite) | From 10.2.0 [CU1] and 10.2.4 onwards | | September 2, 2022 |
+| [ButtonState shapes in Visual Overview fail to hide as expected](xref:KI_ButtonState_shapes_in_Visual_Overview_fail_to_hide_as_expected) | DataMiner Cube 10.4.2425.2536 or lower<br>DataMiner 10.4.8, if Cube client does not have internet access | DataMiner 10.4.8 | August 13, 2024 |
```

Comment:
The version here and on the other lines probably also has to change?

Reply:
This has been adjusted in [commit 29b2a14](https://github.com/SkylineCommunications/dataminer-docs/pull/3567/commits/29b2a14fc4bb0ce70135f976ccd3519bfa273c67).